assets/scss/components/_avatar.scss

```diff
@@ -75,7 +96,7 @@ Class                     | Description
 .avatar {
 	@extend %imgFill;
 	position: relative;
-	background-color: $C_textHint;
+	background-color: $avatarBg; // $C_textHint opaque, #888 with white scrim
```

Comment:
This comment doesn't seem to be correct - `$avatarBg` maps directly to `$C_textHint`